pkg/rancher-desktop/pages/images/add.vue

```diff
@@ -89,7 +90,7 @@ export default {
       this.imageOutputCuller = getImageOutputCuller(command);
     },
     doImageAction({ action, image }) {
-      this.imageToPull = image;
+      this.imageToPull = image.trim();
```

Comment:
Would it make more sense to do the trimming in `ImagesFormAdd` where the even gets sent?
https://github.com/rancher-sandbox/rancher-desktop/blob/6f0582e5675c0c96558795673b28c7192f4cf803/pkg/rancher-desktop/components/ImagesFormAdd.vue#L77

Reply:
sure I will fix it with this issue as its a small change - https://github.com/rancher-sandbox/rancher-desktop/issues/6514